projects/indicators/src/modules/chevron/chevron.component.ts

```diff
@@ -6,15 +6,40 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
   templateUrl: './chevron.component.html',
 })
 export class SkyChevronComponent {
-  @Output()
-  public directionChange = new EventEmitter<string>();
+  @Input()
```

Comment:
Does this need a JSDocs comment to add the public property to the docs? If so, what does this property do. I scanned the existing docs, and I don't see any other `ariaControls` properties to use for comparison. Is this similar to the `ariaRole` properties we have elewhere?

Reply:
@blackbaud-johnly Thanks for the feedback! This is an internal-only component, so it never had any documentation. I went ahead and added jsdocs comments for all the `@Input()`s and `@Output()`s. Thanks!